### ISSUE TITLE
Tier System/Gamification/Badges

### ISSUE THREAD

**Issue by swr5da** [bwang22/logoff#7]
Hi Ben, here are some of our thoughts about internal structure and the tier system:
1. Neighbor - Uses the app, has access to map resources, can post prayer requests, and can add basic info to the map, check in (foursquare), and can hi-five stuff
2. Steward - To become a steward, a neighbor must recruit 40 people to join their squad, write original or retweet 20 prayer request for your neighborhood, mapped or checked in at 20 locations, and pass a short LoGOFF quiz -> That person would unlock "Steward" functions, which would allow you to point your squad toward events and gatherings (Plan events, e.g. Stephen posted his squad to pray at a certain place/to read a certain article etc.), to input more detailed information on to the map, curate neighbor's prayer requests and posts. A steward needs to post at least one prayer request for their neighborhood per week and check into at least 5 places per month and point squad to at least one event per month in order to maintain steward status. 

An active steward is an "Alpha Steward." A steward who does not do those things will become a "Beta Steward." In order to become an "Alpha Steward" again, you have do complete the 4 tasks and maintain  them. 

Basically, you have to keep being an active steward every month or your title will change slightly. If somebody loses steward status, they must complete those 4 tasks again to get it back (won't lose functionality on the app, but will lose title, and must adhere to standards to get it back).

Alpha Steward - Active Steward
Beta Steward - Must complete the 4 tasks in order to become Alpha Steward again

We think it'd be cool to have badges, like an achievement system. You could get a badge for intercession, and become an intercessor if you prayed for places in your neighborhood a certain number of times per week. 

Cool?

**Comment by bwang22** [comment]
Noted. Will try to do this after Maps.
If you have map ideas please post here.
